src/Widgets/ColumnContainerWidget/ColumnContainerWidgetEditingConfig.tsx

```diff
@@ -1,19 +1,73 @@
-import { provideEditingConfig } from 'scrivito'
+import {
+  canEdit,
+  isComparisonActive,
+  provideEditingConfig,
+  Widget,
+} from 'scrivito'
 import { ColumnContainerWidget } from './ColumnContainerWidgetClass'
 import { ColumnsEditorTab } from './ColumnsEditorTab'
 import { ColumnWidget } from '../ColumnWidget/ColumnWidgetClass'
+import { AdvancedEnumEditor } from '../../Components/ScrivitoExtensions/AdvancedEnumEditor'
+import startSvg from './alignment-start.svg'
+import centerSvg from './alignment-center.svg'
+import endSvg from './alignment-end.svg'
+import stretchSvg from './alignment-stretch.svg'
 import Thumbnail from './thumbnail.svg'
 
 provideEditingConfig(ColumnContainerWidget, {
   title: 'Columns',
   thumbnail: Thumbnail,
+  attributes: {
+    alignment: {
+      title: 'Alignment',
+      description: 'Default: Top',
+    },
+  },
+  properties: [
+    [
+      'alignment',
```

Comment:
* [ ] FYI In the sidebar, the alignment setting now takes nearly half the available space. This pushes the most frequently used setting (column layout) down, and hides the two boolean toggles completely. I'd vote for reordering by frequency of use and/or making the alignment setting take less space. (Unless there were UX issues with the previous compact UI?)

Before | After
-- | --
<img width="268" height="760" alt="Image" src="https://github.com/user-attachments/assets/079445ef-429e-4c02-9f14-9c4c988032f9" /> | <img width="268" height="760" alt="Image" src="https://github.com/user-attachments/assets/63851b84-77ec-4a8b-9433-6aa4b0c373a8" />